### PR TITLE
sdl2_net: Add required libraries to pkgconfig on static build

### PR DESCRIPTION
**Pull request by orgads**
See https://github.com/libsdl-org/SDL_net/pull/35

### CODE REVIEW

**Comment by mabrand** [comment]
I think it would be better to do this with a patch instead of a "sed hack".

**Comment by orgads** [comment]
Done. Source: https://github.com/scummvm/dockerized-bb/blob/master/toolchains/mxe/packages/sdl2_net/sdl2_net-1-pkg-config.patch

**Comment by mabrand** [comment]
Sorry about nitpicking, but could you please mention the origin of the patch in the commit message of the patch in the .patch file? Such as "Taken from <URL>".

**Comment by orgads** [comment]
No problem at all. Done.

File: src/sdl2_net-1-pkg-config.patch [hunk 1]
```diff
@@ -0,0 +1,80 @@
+From e2782ccaa642202e50aebefe4107e4313f6bbb53 Mon Sep 17 00:00:00 2001
+From: Cameron Cawley <ccawley2011@gmail.com>
+Date: Sat, 17 Jul 2021 15:56:53 +0100
+Subject: [PATCH] Include required libraries in pkg-config output
+
+---
```
Comment (mabrand) [inline src/sdl2_net-1-pkg-config.patch:6]: Please include "Taken from.." here.
Also, the stats aren't necessary. The script tools/patch-tool-mxe can be used to "normalize" patch format.

Comment (orgads) [reply src/sdl2_net-1-pkg-config.patch]: Done.

**Pull request merged**